Directory.Build.props

```diff
@@ -5,7 +5,7 @@
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <OutputDrop>$(TF_BUILD_BINARIESDIRECTORY)</OutputDrop>
-    <NoWarn>$(NoWarn),1570,1572,1573,1574,1591,1701</NoWarn>
+    <NoWarn>$(NoWarn),1570,1572,1573,1574,1591,1701,CA0068</NoWarn>
```

Comment:
Do you know why we get this warning? I see .pdbs next to all of our assemblies...

Reply:
Yeah - probably because we don't create "full" pdbs. FxCop is out of date with the new project/debug types. This suppression doesn't work anyway, so I'll remove it.